packages/material-ui-codemod/src/v0.15.0/import-path.test.js

```diff
@@ -22,7 +22,7 @@ describe('@material-ui/codemod', () => {
         );
 
         const expected = read('./import-path.test/expected.js');
-        expect(trim(actual)).to.equal(trim(expected), 'The transformed version should be correct');
+        expect(trim(actual.replace(/\r\n/g, '\n'))).to.equal(trim(expected), 'The transformed version should be correct');
```

Comment:
Instead of repeating the replacement, maybe we should move it to the `read()` method and make it a shared helper for the codemod tests?

Reply:
Sure - done.